plugins/github-actions/src/components/WorkflowRunDetails/WorkflowRunDetails.tsx

```diff
@@ -55,6 +54,9 @@ const useStyles = makeStyles<Theme>(theme => ({
   title: {
     padding: theme.spacing(1, 0, 2, 0),
   },
+  breadcrumb: {
+    margin: theme.spacing(0, 0, 3, 0),
+  },
```

Comment:
This is fine, but it may be even easier to wrap the breadcrumbs in `<Box mb={3}>...</Box>` (where the Box is from material-ui; mb = margin-bottom). It's a great utility component that makes some layout stuff easier. Like a div on steroids